packages/manager/modules/anthos/src/dashboard/general-information/software-update/software-update.html

```diff
@@ -0,0 +1,86 @@
+<div>
+    <oui-back-button
+        href="{{:: $ctrl.generalInformationLink }}"
+        aria-label="Aria label for heading"
+    ></oui-back-button>
+    <oui-header
+        class="mb-0"
+        data-heading="{{:: 'anthos_tenant_dashboard_general_information_software_update_title' | translate}}"
+        data-description="{{:: 'anthos_tenant_dashboard_general_information_software_update_description' | translate}}"
+    ></oui-header>
+    <form name="softwareUpdateForm" data-ng-submit="$ctrl.updateSoftware()">
+        <div class="oui-tile__body">
+            <div class="oui-datagrid-responsive">
+                <table class="oui-datagrid">
```

Comment:
**question**: any particular reason for not using oui-datagrid?

Reply:
Found strange to mix both `oui-tile__body` plus `oui-datagrid*` (morever there's only one column).

Reply:
@anooparveti I couldn't find a simple way to hide the pagination "natively"